Makefile

```diff
@@ -47,6 +47,12 @@ PYTHON_TARGET := 2.7
 REQUIREMENTS := test-requirements.txt requirements.txt
 PIP_OPTIONS := $(ST2_PIP_OPTIONS)
 
+PYLINT_CONCURRENCY := $(PYLINT_CONCURRENCY)
```

Comment:
Why this line? I thought environment variables were considered normal variables within makefiles?

Reply:
You are correct, I believe it's not needed, but I've done it because we also do that a couple of lines above.

Having said that, I can remove it.